### PR TITLE
TT-7: GovWales Outcomes

### PR DESCRIPTION
**Pull request by markheathy**
What
----

Outcome page theming and customisation.

How to review
-------------

Describe the steps required to review and test the changes.

Links
-----

Add links to any relevant [Trello cards](https://trello.com/b/9Pvq5z2n/govuk-coronavirus-services-team-doing-board),
Zendesk tickets or Sentry issues.

### CODE REVIEW

File: app/views/layouts/application.html+govwales.erb [hunk 1]
```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
```
Comment (rickclare) [inline app/views/layouts/application.html+govwales.erb]: We were hoping to get away without having to change this main layout file.  If we have a variant, it's going to make it difficult to retrofit any changes that GDS make on there side.

Can you give us more information about what's changed in this variant, @markheathy ?

**Review by rickclare — COMMENTED** [review]
There's a fair few "variants" created here, @markheathy  If that's necessary, then that's OK, but the more variants we have to have, the less easy it will be to retrofit future GDS changes.  

Are there any compromises (or CSS "hacks") that we can make to avoid having so many extra variant files?

File: config/locales/cy.yml [hunk 2]
```diff
@@ -26,6 +26,13 @@ cy:
       href: https://llyw.cymru/
       alt: Llywodraeth Cymru
       text: Hafan
+  govwales_breadcrumbs:
+    start_again_text: Start again
```
Comment (adamccfc) [inline config/locales/cy.yml]: Might be worth adding a `TODO:` comment here to remind us to add the Welsh translations?

**Review by adamccfc — CHANGES_REQUESTED** [review]

File: app/views/layouts/application.html+govwales.erb (hunk 1 above)
Comment (markheathy) [reply app/views/layouts/application.html+govwales.erb]: The breadcrumb position has been moved above the title block @rickclare to fit WG UX.

File: config/locales/cy.yml (hunk 2 above)
Comment (markheathy) [reply config/locales/cy.yml]: We got the same in the Welsh 'CY' version of the config file.

File: app/views/layouts/application.html+govwales.erb (hunk 1 above)
Comment (markheathy) [reply app/views/layouts/application.html+govwales.erb]: Its a lift and shift, not a radical structural change - Element Ordering.

File: config/locales/cy.yml (hunk 2 above)
Comment (adamccfc) [reply config/locales/cy.yml]: Yeah could even just add CY like we have done else where just to distinguish the two to make sure it is being translated?

**Review by rickclare — APPROVED** [review]
Approved

File: config/locales/cy.yml (hunk 2 above)
Comment (markheathy) [reply config/locales/cy.yml]: These will be addressed during translation.

**Review by adamccfc — APPROVED** [review]

**Pull request merged**